example-sites/angular/src/index.html

```diff
@@ -18,7 +18,7 @@
 		};
 	</script>
   <script src="https://media.evolv.ai/releases/latest/scout.min.js"
-          data-evolv-environment="ad4159c6e7"
+          data-evolv-environment="xxxxxxxxx"
           data-evolv-auto-load="false">
```

Comment:
Autoload should not be disabled.
```suggestion
          >
```